README.md

```diff
@@ -76,7 +76,7 @@ The Community Committee is an autonomous committee that collaborates alongside t
 * [bnb](https://github.com/bnb) - **Tierney Cyren** &lt;hello@bnb.im&gt; - **Community Committee Chair**
 * [chowdhurian](https://github.com/chowdhurian) - **Manil Chowdhury** &lt;manil.chowdhury@gmail.com&gt;
 * [dshaw](https://github.com/dshaw) - **Dan Shaw** &lt;dshaw@dshaw.com&gt;
-* [hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds** &lt;tracyhinds@gmail.com&gt;
+* [hackygolucky](https://github.com/hackygolucky) - **Tracy Hinds** &lt;tracyhinds@gmail.com&gt; **Community Committee Director**
```

Comment:
A nit: in some cases, we have dashes before statuses, in some we have not. Maybe it is worth to unify?

Reply:
Happy to do that in another PR 👍